cpp/common/src/codingstandards/cpp/VariablyModifiedTypes.qll

```diff
@@ -0,0 +1,214 @@
+import cpp
+
+/**
+ * A declaration involving a variably-modified type.
+ */
+class VmtDeclarationEntry extends DeclarationEntry {
```

Comment:
I think we're only interested in variable declarations, right?
```suggestion
class VmtDeclarationEntry extends VariableDeclarationEntry {
```

Reply:
This actually causes the query to exclude the typedef results, since those also use `DeclarationEntry`s

Reply:
Ah, makes sense.